themes/docsy/layouts/partials/page-meta-lastmod.html

```diff
@@ -1 +1,3 @@
-{{ T "post_last_mod"}} {{ .Lastmod.Format .Site.Params.time_format_default }}{{ with .GitInfo }}: <a  href="{{ $.Site.Params.github_repo }}/commit/{{ .Hash }}">{{ .Subject }} ({{ .AbbreviatedHash }})</a>{{end }}
+{{ if and (.GitInfo) (.Site.Params.github_repo) -}}
```

Comment:
I'm modifying docsy theme files and we already have quite a lot of modifications in the theme (in vendored fiels basically) but it will all go away once we merge #284.